store/src/java/com/zimbra/qa/unittest/TestRemoteImapNotifications.java

```diff
@@ -275,15 +274,21 @@ public void testDeleteTagNotificationActiveFolder() throws Exception {
 
         Flags flags = info.getPermanentFlags();
         assertTrue(flags.contains(new Atom(tagName)));
+        Map<Long, MessageData> mdMap = connection.fetch("1:*", "(FLAGS)");
+        Flags msgFlags = mdMap.get(1L).getFlags();
+        assertTrue(msgFlags.contains(new Atom(tagName)));
```

Comment:
I'd prefer the assert variants which leave a clear message in the reports when they fail.
Perhaps we need to discuss as a team whether I should carry on saying this or whether it isn't important enough?